docs/source/GetStartedGuide.rst

```diff
@@ -93,6 +104,12 @@ Note that `python` and `pip` should point to the Python interpreter version 2.7.
 If you're having trouble to install, you can try again and use the command line
 utilities `python2.7` and `pip2.7`.
 
+Please note that some Avocado functionality may be implemented by
+optional plugins.  To install say, the HTML report plugin, run::
+
+    cd optional_plugins/html
```

Comment:
Actually, starting with 1303c8658b3ec0c4999869bd314efaa30311df11, `make develop` now goes through the optional plugins (in tree), while `make link` also goes through the external plugins in the parent directory.